### DISCUSSION TITLE
Option to turn off volume control / set to fixed value for particular player

### DISCUSSION THREAD

**Discussion by mkrysmann** [music-assistant/support#1364] [Feature requests and ideas]
For some configurations where player acts as pre-amp and then power amp is used to control volume it would be useful to be able to set volume on fixed level and hide volume slider from main UI. Then it won't be sound level differences between different pre-amp sources (Music Assistant and f.ex. AppleTV), all volume would be controlled on separate device.

**Comment by marcelveldt** [comment]
We're even going to do better where you can attach a volume control to a player in MA. This volume control can then be "fixed", the players own volume control or a HA entity. For example control the volume slider of your receiver.

**Comment by OzGav** [comment]
This has been implemented in MA 2.4

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
